examples/bpi-monolith-test-suite/bpi/components/zkmock/blockchain/blockchain.js

```diff
@@ -0,0 +1,89 @@
+import { solc } from "solc";
```

Comment:
in future we could probably make small utils for all reusable files

btw, why is this one js and one above ts?